app/actions/local/notification.ts

```diff
@@ -28,6 +23,8 @@ export const updatePostSinceCache = async (serverUrl: string, notification: Noti
         }
         return {};
     } catch (error) {
+        // eslint-disable-next-line no-console
+        console.log('Failed updatePostSinceCache', error);
```

Comment:
Should we start using a "custom log function" so we can skip the `// eslint-disable-next-line no-console` comment? What I mean is having defined somewhere in utils a log function as:
```
function log(args) {
    // eslint-disable-next-line no-console
    console.log(...args);
}
```
That way we also keep separated actual logs from debug messages. That way we also are able to add in the future a more centralized way to maybe send the log messages to sentry, add some extra formatting (like log level), or things like that.